packages/fluentui/docs/src/examples/components/Card/BestPractices/CardBestPractices.tsx

```diff
@@ -5,9 +5,16 @@ import ComponentBestPractices from '../../../../components/ComponentBestPractice
 const doList = [
   'Limit use of `aria-roledescription` property to clarifying the purpose of card, or to providing a more specific description of a widget.',
   'Use `cardBehavior` for disabled cards, so it would not be focusable.',
```

Comment:
isn't `cardBehavior` default? why would the consumer need to change the behavior to make disabled work correctly?

Reply:
Oh, yes, that makes sense